doc/sphinx/source/conf.py

```diff
@@ -263,8 +263,10 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-  ('index', 'ESMValTool_Users_Guide.tex', u'ESMValTool User\'s and Developer\'s Guide',
-   u'Veronika Eyring, Axel Lauer, Mattia Righi, Martin Evaldsson et al.', 'manual'),
+    ('index', 'ESMValTool_Users_Guide.tex',
+     u'ESMValTool User\'s and Developer\'s Guide',
+     u'Veronika Eyring, Axel Lauer, Mattia Righi, Martin Evaldsson et al.',
+     'manual'),
```

Comment:
This author list is outdated.
Maybe you could add the current core development team or simply "The ESMValTool Core Development Team".